src/main/scala/com/cognite/sdk/scala/v1/dataPoints.scala

```diff
@@ -43,3 +43,7 @@ final case class QueryRangeById(
     start: String,
     end: String
 )
+
+final case class QueryByAggregate(id: Long, start: String, end: String, granularity: String, aggregates: Seq[String])
```

Comment:
`QueryAggregatesById` since you'll also need to add a `QueryAggregatesByExternalId` (and aggregates since there may be more than one).